Resources/Prototypes/_EE/Entities/Mobs/Customization/Markings/xelthia.yml

```
@@ -0,0 +1,92 @@
+  id: XelthiaBackspikes
+  bodyPart: Chest, RArm, LArm, Head
+  markingCategory: Overlay
+  speciesRestriction: [Xelthia]
+  coloring:
+    default:
+      type:
+        !type:SimpleColoring
+          color: "#FFFFFF"
+  sprites:
+  - sprite: _EE/Mobs/Customization/xelthia_parts.rsi
+    state: backspikes
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)